modules/integration/src/test/scala/scala/cli/integration/BloopTests.scala

```diff
@@ -134,36 +135,38 @@ class BloopTests extends ScalaCliSuite {
     }
   }
 
-  test("Restart Bloop server while watching") {
-    TestUtil.withThreadPool("bloop-restart-test", 2) { pool =>
-      val timeout = Duration("90 seconds")
-      val ec      = ExecutionContext.fromExecutorService(pool)
-
-      def content(message: String) =
-        s"""object Hello {
-           |  def main(args: Array[String]): Unit =
-           |    println("$message")
-           |}
-           |""".stripMargin
-      val sourcePath = os.rel / "Hello.scala"
-      val inputs = TestInputs(
-        sourcePath -> content("Hello")
-      )
-      inputs.fromRoot { root =>
-        val proc = os.proc(TestUtil.cli, "run", "--power", "--offline", "-w", ".")
-          .spawn(cwd = root)
-        val firstLine = TestUtil.readLine(proc.stdout, ec, timeout)
-        expect(firstLine == "Hello")
-
-        os.proc(TestUtil.cli, "--power", "bloop", "exit")
-          .call(cwd = root)
-
-        os.write.over(root / sourcePath, content("Foo"))
-        val secondLine = TestUtil.readLine(proc.stdout, ec, timeout)
-        expect(secondLine == "Foo")
-
-        proc.destroy()
+  if (!Properties.isMac || !TestUtil.isNativeCli || !TestUtil.isCI)
+    // TODO make this pass reliably on Mac CI
```

Comment:
This should have an issue otherwise we might never get back to it.

Reply:
Created a ticket: https://github.com/VirtusLab/scala-cli/issues/2517